app/controllers/staff/activity_forms_controller.rb

```diff
@@ -95,6 +109,10 @@ def activity_params
       :aid_type)
   end
 
+  def level
+    params.require(:activity).permit(:level)["level"]
```

Comment:
Is this strong parameters on a single param?  Is this instead of adding `:level` to the giant `permit` on line 106? I would love to understand this better.

Reply:
It is.

I started by adding it there but found I could change the `level` of an activity again since the `update_activity_attributes_except_dates` is run on every update. It takes all `activity_params` and blindly assigns them. To me that needed a refactor and I decided to work around it for now.